src/firefly/js/visualize/task/PlotHipsTask.js

```diff
@@ -496,13 +496,13 @@ export function matchHiPSToImage(pv, hipsPVidAry) {
         Object.entries(attributes).forEach( (entry) => dispatchAttributeChange(id, false, entry[0], entry[1]));
         dispatchAttachLayerToPlot(ImageOutline.TYPE_ID, id);
         dispatchChangeCenterOfProjection({plotId:id, centerProjPt:wpCenter});
-
+        dispatchChangeHiPS( {plotId:id,  coordSys:plot.imageCoordSys});
```

Comment:
There is a slight problem here.  HiPS only supports `galactic` and `j2000`.  it is possible that that `plot.imageCoordSys` might be something like `b1950`.  So this line should be wrapped in a `if` to check if it is one of those two coordinate system. If not is should just keep the existing coordinate system.

Reply:
I added the if block as you suggested.  Thanks!